servo-core/src/main/java/com/netflix/servo/stats/StatsBuffer.java

```diff
@@ -93,7 +93,7 @@ public void reset() {
    * Record a new value for this buffer.
    */
   public void record(long n) {
-    values[count++ % size] = n;
+    values[(int) Long.remainderUnsigned(count++, size)] = n;
```

Comment:
Integer.remainderUnsigned seems more appropriate here and would not need a cast.